src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let sum = days * 40;
+
+  if (days >= 3 && days <= 6) {
```

Comment:
Consider using ternary operator for better readability and simplicity 🤔

Reply:
Done